packages/web-react/src/components/Toast/UncontrolledToast.tsx

```diff
@@ -4,30 +4,32 @@ import ToastBar from './ToastBar';
 import { useToast } from './useToast';
 import { UncontrolledToastProps } from '../../types';
 
-const defaultProps = {
-  isDismissible: true,
-};
-
 const UncontrolledToast = (props: UncontrolledToastProps) => {
-  const propsWithDefaults = { ...defaultProps, ...props };
-  const { alignmentX, alignmentY, closeLabel, hasIcon, isDismissible, ...restProps } = propsWithDefaults;
-  const { hide, color, iconName, message, id, isOpen } = useToast();
+  const { alignmentX, alignmentY, closeLabel, ...restProps } = props;
+  const { hide, queue } = useToast();
 
   return (
     <Toast alignmentX={alignmentX} alignmentY={alignmentY}>
-      <ToastBar
-        id={id}
-        closeLabel={closeLabel}
-        color={color}
-        hasIcon={hasIcon}
-        iconName={iconName}
-        isDismissible={isDismissible}
-        onClose={hide}
-        isOpen={!!isOpen && !!message}
-        {...restProps}
-      >
-        {message}
-      </ToastBar>
+      {queue.map((item) => {
+        const { color, iconName, id, isOpen, message, hasIcon, isDismissible } = item;
+
+        return (
+          <ToastBar
+            key={id}
+            id={id}
+            closeLabel={closeLabel}
+            color={color}
+            hasIcon={hasIcon}
+            iconName={iconName}
+            isDismissible={isDismissible}
+            onClose={() => hide(id)}
+            isOpen={!!isOpen && !!message}
+            {...restProps}
```

Comment:
Please, move the rest props at the start.

Reply:
moved